### PR TITLE
Initial skeleton for News.

### PR DESCRIPTION
**Pull request by nessita**
Part of #273

### CODE REVIEW

File: templates/news/update.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{% extends "base.html" %}
+{% load i18n %}
+
+{% block content %}
+<h1>{% if news.pk %}{% translate "Edit News" %}{% else %}{% translate "Create News" %}{% endif %}</h1>
```
Comment (vinniefalco) [inline templates/news/update.html:5]: What's "translate" ?

Comment (nessita) [reply templates/news/update.html:5]: `{% translate %}` is [the Django template tag](https://docs.djangoproject.com/en/4.2/topics/i18n/translation/#translate-template-tag) that allows us to mark literal strings for potential/future translation. I'm not sure if this is something we'd want or not, it certainly does not hurt nor force us to translate anything yet, but if there are future plans to show the site in other languages, this is the way.

Comment (vinniefalco) [reply templates/news/update.html:5]: oh cool

**Comment by nessita** [comment]
> @nessita OK to close this? Looks like it's mostly a duplicate of #345

Yes, thank you for the ping! I'm almost done with the part 2 of this, to allow CUD operations.

**Pull request closed without merging**